### ISSUE TITLE
screenreader did not jump inside rendered ScaleTable

### ISSUE THREAD

**Issue by HejHG** [telekom/scale#2201]
**Scale Version**
 "@telekom/scale-components": "3.0.0-beta.142",
 "@telekom/scale-components-react": "3.0.0-beta.142"

**Framework and version**
"react": "18.2.0",

**Current Behavior**
ScaleTable is rendered dynamically and can be accessed using the tab key, as can the individual cells in the table.
The screen reader in turn finds the table, but cannot jump into it, so the individual cells cannot be read out.
Without ScaleTable-tag the whole table can be read with screanreader
 
**Expected Behaviour**
with screenreader I can jump inside table an can read every cell

**Code Reproduction**
return ( 
<div>
<ScaleTable>
      <table >
...
</table>
</ScaleTable>
<div>);

**Desktop (please complete the following information):**
MacOs 14.0, also an windows terminals for Accessibility tests

**Smartphone (please complete the following information):**
n.r.

**Additional context**
n.r.

**Comment by felix-ico** [comment]
Hi @HejHG I'm not sure I understand the issue, i quickly tested the scale-table here https://telekom.github.io/scale/iframe.html?id=components-table--standard&args=&globals=locale:en;colorMode:light&viewMode=story using Edge & JAWS and the table and all its contents were announced by the screen reader. I was able to navigate the individual elements in the table with the arrow keys.

Or did you mean something else?

**Comment by amir-ba** [comment]
@HejHG can you tell us if this problem still exists?

**Comment by HejHG** [comment]
@amir-ba : hi, the problem no longer exists because I have adapted the solution. If I remember correctly, I had defined the attribute role={‘grid’} on the ScaleTable tag and then the ScreenReader no longer wanted to jump into the table. The attribute is now on the table-tag and now it works.
